plugins/SEO-Tool/js/api/lighthouse.js

```diff
@@ -0,0 +1,54 @@
+function detailedLink(url, strategy) {
+	return 'https://googlechrome.github.io/lighthouse/viewer/?psiurl=' + url + '&strategy=' + strategy;
+}
+
+async function lighthouseApi(link, apiKey) {
+	const strategy = ['mobile', 'desktop'], pageData = {};
+
+	const proms = strategy.map(async (val, ind) => {
+		const api = 'https://www.googleapis.com/pagespeedonline/v5/runPagespeed', parameters = {
+			url: encodeURIComponent(link),
+			key: apiKey,
+			category: ['performance', 'accessibility', 'best-practices', 'seo', 'pwa'],
+			strategy: val 
+		}, url = setUpQuery(link, api, parameters);
+
+
+		await axios.get(url)
+			.then(response => {
```

Comment:
After await, there's no need to use .then clause. You can directly write the next line of code. Use try..catch for errors.